feature/gnmi/telemetry_npu_asic_counters_test/tests/README.md

```diff
@@ -0,0 +1,176 @@
+# gNMI-1.7: Telemetry: NPU/ASIC Counters
+
+## Summary
+
+Validate the supported OC Pipeline Counters.
+
+## Procedure
+
+Note: New telemetry model is still being defined.
+
+*   Connect ATE port-1 to DUT port-1. ATE port-2 to DUT port-2. ATE port-3 to DUT port-3.
+*   OuterDstIP_1, OuterSrcIP_1, OuterSrcIP_2: IPinIP outer IP addresses.
+*   InnerDstIP_1, InnerSrcIP_1: IPinIP inner IP addresses.
+*   ATEPort2IP: testbed assigned interface IP to ATE port 2.
+*   ATEPort3IP: testbed assigned interface IP to ATE port 3.
+*   Connect a gRIBI client to the DUT, make it become leader and inject the following,
+
+    NHG#1 --> NH#1 {next-hop: ATEPort2IP, network-instance:DEFAULT}
+    OuterDstIP_1/32  --> NHG#1.
+
+*   Send IPinIP traffic to OuterDstIP_1. Validate that ATE port-2 receives the IPinIP traffic.
+
+*   Packet Counters - Interface-Block
+
+    *   Subscribe to the following, and capture values of the 4 leaves (1).
+        * /packet/interface-block/in-pkts
+        * /packet/interface-block/out-pkts
+        * /packet/interface-block/in-bytes
+        * /packet/interfac-block/out-bytes
+    *   Wait for 30 secs and capture the values of 4 leaves(2). 
+    *   Calculation of step 2 - step 1 leaf values. 
+        * 30*pps = in-pkts and out-pkts
+        * 30*pps*300 = in-bytes and out-bytes
+    * Stop traffic.
+
+*   Packet Counters - Host-Block
+
+    *   Subscribe to the following, and capture the values of the 4 leaves(1),
+        * /packet/host-interface-block/in-pkts
+        * /packet/host-interface-block/out-pkts
+        * /packet/host-interface-block/in-bytes (TBD)
+        * /packet/host-interface-block/in-bytes (TBD)
+    *   Define variance = 0.99
+    *   Send ping packets to ATEPort2IP with count 1000. Wait for 20 secs.
+    *   Capture the in-pkts and out-pkts leaves(2).
+    *   Calculation of step 2 - step 1 leaf values.
+        * Step 2 - Step 1 >= 1000
+        * Step 2/Step 1 >= variance
+    *   Verify in-bytes and out-bytes (TBD)
+
+*   Packet Countes - Queueing Block
+
+    *   Subscribe to the following, and capture the values of the 4 leaves(1),
+        * /packet/queueing-block/in-pkts(TBD)
+        * /packet/queueing-block/in-bytes(TBD)
+        * /packet/queueing-block/out-pkts,
+        * /packet/queueing-block/out-bytes
+    *   Define variance = 0.98.
```

Comment:
why 98% ? why not 99% as previous? why not 99.9%

Reply:
Had written the calculation part for out-pkts as an exact match for expected packet count. The expectation for variance is a redundant item here, have modified the variance value to 0.999.